frame/elections-phragmen/src/benchmarking.rs

```diff
@@ -459,7 +459,7 @@ benchmarks! {
 	#[extra]
 	election_phragmen_v {
 		let v in 4 .. 16;
-		let fixed_c = MAX_CANDIDATES / 10;
+		let fixed_c = T::MaxCandidates::get() / 10;
```

Comment:
You should be using smaller values for test, similar to the old code. 

I would do that by `let max_voters = if cfg!(test) { TEST_MAX_VOTERS } else { T::MaxVoters::get() } `

Reply:
what I wrote was bullshit :p